Competition2017/src/competition/subsystems/climbing/RopeAlignerSubsystem.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.climbing;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class RopeAlignerSubsystem extends BaseSubsystem {
+    private static Logger log = Logger.getLogger(RopeAlignerSubsystem.class);
+    
+    public final XCANTalon intakeMotor;
```

Comment:
I agree the extra class _makes_ it safe, but I disagree that its "very little effort".  I personally find the extra classes cumbersome and too enterprise-fizz-buzz-ey for the complexity of code we're writing. That said, if people want to go that route I won't stand in anyone's way but I support the option not to have to write that extra boiler plate code for simple systems.